test/e2e/specs/editor/various/keyboard-navigable-blocks.spec.js

```diff
@@ -12,8 +12,9 @@ test.use( {
 } );
 
 test.describe( 'Order of block keyboard navigation', () => {
-	test.beforeEach( async ( { admin } ) => {
+	test.beforeEach( async ( { admin, editor } ) => {
 		await admin.createNewPost();
+		await editor.openDocumentSettingsSidebar();
```

Comment:
The test expects the document settings sidebar to be open, otherwise keyboard navigation fails.